src/rete/controls/LabeledInputControl.tsx

```diff
@@ -7,10 +7,28 @@ export class LabeledInputControl extends Classic.InputControl<'number'> {
         change?: () => void,
         readonly?: boolean
     ) {
-        super('number', { initial: value, readonly: Boolean(readonly), change })
+
+        super('number', { initial: value, readonly: Boolean(readonly), change: change })
+    }
+
+    setValue(newValue: number): void {
```

Comment:
This seems to get called whenever the value changes from _any_ source - that means that this is called for manual text entry as well. So this breaks when I try to type the value in instead of using the arrows; it'll overwrite what I typed in.

Reply:
Also, it seems like this is overwriting something from the base class; we'll probably have to call the inherited version of setValue() somewhere to make sure it performs properly. Right now, changes to the field don't reflect in the editor unless the node's connections get changed, which is telling me the `change` function passed in through the constructor isn't getting called (I'm assuming the base class for `InputControl` calls `change` inside setValue).